proposals/FileSystemObserver.md

```diff
@@ -374,7 +374,7 @@ await observer.observe(directoryHandle, { recursive: true });
 // corresponding to `directoryHandle`, then `touch file.txt`)
 ```
 
-User agents may also use the `"unsupported"` `FileSystemChangeType` to explicitly indicate that change types are not supported.
+User agents may also use the `"unknown"` `FileSystemChangeType` to indicate that a change has occurred but the type of change is unknown or is not to be specified to the site. This could mean that the change type does not belong to any other values defined in `FileSystemChangeType` due to cross-platform differences, or a change has occurred while the site was not fully active (i.e. site in back/forward cache becomes fully active again).
```

Comment:
I really don't think you should say it could be a change type that doesn't belong to any other values. Currently unknown is being used to say it "unknown" if zero or more events have occurred, so that the developer will trigger a poll.

Sending out "unknown" for OS events that are specific to some platform would be a confusing signal to developers.

Reply:
Rephrased this section into a new section about interactions with BFCache, as it is not related to cross-platform differences.